tests/net/all/prj.conf

```diff
@@ -106,8 +106,6 @@ CONFIG_NET_CONTEXT_SYNC_RECV=y
 CONFIG_NET_CONTEXT_CHECK=y
 
 # SLIP
-CONFIG_NET_SLIP_TUN=n
-CONFIG_NET_SLIP_TAP=y
```

Comment:
The purpose of the tests/net/all is to enable all possible options so that we can catch various compile errors early, so I wonder why we could not keep these options?